packages/babel-helpers/src/helpers/usingCtx.ts

```diff
@@ -2,17 +2,17 @@
 
 const enum DisposeFlag {
   // set when the dispose function is a [Symbol.asyncDispose] method
-  ASYNC_FLAG = 1,
-  // alias of AWAIT_FLAG: set when the resource is an `await using` disposable
-  ASYNC_MASK = 2,
+  ASYNC_DISPOSE = 1,
+  // alias of AWAIT_USING: set when the resource is an `await using` disposable
+  ASYNC_DISPOSE_MASK = 2,
 }
 
 const enum DisposeKind {
-  SYNC = 0,
+  USING_DISPOSE = 0,
   // AWAIT_FLAG
-  AWAIT_SYNC = 2,
-  // AWAIT_FLAG | ASYNC_FLAG
-  AWAIT_ASYNC = 3,
+  AWAIT_USING_DISPOSE = 2,
+  // Flag.AWAIT_USING | Flag.ASYNC_DISPOSE
+  AWAIT_USING_ASYNC_DISPOSE = 3,
```

Comment:
I renamed the enums so that hopefully two-year-later me will not get lost when reviewing the helper.